enterprise/internal/github_apps/store/store.go

```diff
@@ -90,16 +96,30 @@ func (s *gitHubAppsStore) Create(ctx context.Context, app *ghtypes.GitHubApp) (i
 	if err != nil {
 		return -1, err
 	}
+
+	baseURL, err := url.Parse(app.BaseURL)
+	baseURL = extsvc.NormalizeBaseURL(baseURL)
 	domain := app.Domain
 	if domain == "" {
 		domain = types.ReposDomain
 	}
 
+	// Check if a GitHub App already exists for GitHub instance if it's a batches domain App
```

Comment:
In this case, I feel a comment about the intent of this code might be more helpful than a comment about what's actually happening here, since I think the code itself is pretty straightforward to read. 🙂

```suggestion
	// We enforce that GitHub Apps created in the "batches" domain are for unique instance URLs.
```

Reply:
Ah yea, this is much more clear. Thanks!